client/lib/media/utils.js

```diff
@@ -37,7 +37,7 @@ const REGEXP_VIDEOPRESS_GUID = /^[a-z\d]+$/i;
  *
  * @param  {Object} media   Media object
  * @param  {Object} options Optional options, accepting a `photon` boolean,
- *                          `maxWidth` pixel value, or `size`.
+ *                          `maxWidth` pixel value, `resize` string, or `size`.
  * @return {string}         URL to the media
  */
 export function url( media, options ) {
```

Comment:
i wonder if we could consolidate this with `lib/resize-image-url`?

Reply:
This function already uses `lib/resize-image-url`. What sort of consolidation did you have in mind?

Reply:
yeah, i see that now. hrm.